app/src/main/java/fr/free/nrw/commons/upload/UploadService.java

```diff
@@ -162,7 +162,7 @@ public void queue(int what, Contribution contribution) {
                         .subscribeOn(ioThreadScheduler)
                         .observeOn(mainThreadScheduler)
                         .subscribe(aLong->{
-                            contribution.set_id(aLong);
+                            contribution.setPageId(String.valueOf(aLong));
```

Comment:
Not sure about this at all, at this point a contribution can't have a pageID as it doesn't exist on the servers, we try and save it with nothing(?) set as pageId? I am pretty sure `@AutoGenerate` has been removed so that won't work